float32 maxT
float32 minT